crates/ra_hir/src/ty/tests.rs

```diff
@@ -21,19 +21,57 @@ use crate::{
 // update the snapshots.
 
 #[test]
-fn infer_for_loop() {
+fn infer_try() {
     let (mut db, pos) = MockDatabase::with_position(
         r#"
 //- /main.rs
-struct Vec<T> {}
-impl<T> Vec<T> {
-    fn new() -> Self { Vec {} }
-    fn push(&mut self, t: T) { }
+
+fn test() {
+    let r: Result<i32, u64> = Result::Ok(1);
+    let v = r?;
+    v<|>;
 }
 
-impl<T> ::std::iter::IntoIterator for Vec<T> {
-    type Item=T;
+//- /std.rs
+
+#[prelude_import] use ops::Try;
```

Comment:
I don't think these `prelude_import`s do anything: `prelude_import` can annotate one glob import, which will then be imported as prelude. So the below glob import is actually used as prelude; this attribute does nothing.